Extensions/DataSources/azure-devops-extension.json

```diff
@@ -0,0 +1,27 @@
+{
+  "manifestVersion": 1,
+  "id": "DataSources",
+  "publisher": "ms-vscs-rm",
+  "scopes": [
+    "vso.project",
+    "vso.serviceendpoint_query",
+    "vso.gallery"
```

Comment:
this is fix for the issue we are facing to access file in others machine?

Reply:
Gyan is still trying with vso.gallery but didn't work